test/test_va_api_fixture.cpp

```diff
@@ -189,6 +236,56 @@ void VAAPIFixture::doFillConfigAttribList()
     EXPECT_EQ(m_configAttribList.size(), m_vaConfigAttribs.size());
 }
 
+void VAAPIFixture::doTestCreateImage(const uint32_t& currentFmt,
+		const std::pair<uint32_t, uint32_t>& currentResolution)
+{
+    VAStatus status;
+    VAImageFormat currentImageFormat;
+
+    currentImageFormat.fourcc = currentFmt;
+
+    status = vaCreateImage(m_vaDisplay, &currentImageFormat,
+		 currentResolution.first, currentResolution.second, &m_image);
+
+    if (status == VA_STATUS_SUCCESS)
+	ASSERT_ID(m_image.image_id);
+    else
+	EXPECT_TRUE(
```

Comment:
Hard tabs here instead of soft. Lines 253 & 254.

Reply:
Fixed.